packages/shared/lib/models/NangoConfig.ts

```diff
@@ -41,25 +35,17 @@ export interface NangoV2IntegrationContents {
     'post-connection-scripts'?: string[];
 }
 
-export interface NangoV2Integration {
-    // providerConfigKey
-    [key: string]: NangoV2IntegrationContents;
-}
+export type NangoV2Integration = Record<string, NangoV2IntegrationContents>;
 
-export interface NangoModelV1 {
-    // modelName
-    [key: string]: {
-        // field name
-        [key: string]: string | Record<string, string>;
-    };
-}
+export type NangoModelV1 = Record<string, Record<string, string | Record<string, string>>>;
```

Comment:
Having these comments is/was pretty useful

Reply:
👌🏻  reverted this part